onnxscript/converter.py

```diff
@@ -542,18 +545,18 @@ def translate_expr(
             raise ValueError(
                 self.message(node, f"Unsupported expression type {type(node)!r}.")
             )
-        if isinstance(r, ConverterExpression):
+        if isinstance(r, Variable):
             return r
         if isinstance(r, tuple):
             callee, args, attrs = r
             target = "tmp" if target is None else target
             assert isinstance(target, str)
             result = self.generate_unique_name(target)
             self.emit([result], callee, args, attrs)
-            return ConverterExpression(result, ConverterExpressionKind.ANY)
-        return ConverterExpression(r, ConverterExpressionKind.ANY)
+            return Variable(result)
+        return Variable(r)
```

Comment:
Looks like this line needs a test

Reply:
Good catch, it is not needed any longer, since the translate_ method(s) that were returning a string have been updated to return a Variable.